app/views/posts/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h2><%= @post.title %></h2>
+<h3><%= @post.user.email %></h3>
+<h3><%= @post.created_at.strftime("%m/%d/%y")%></h3>
+<div>
+  <p class="text"><%= @post.text %></p>
+  <%= link_to "Edit Post", edit_post_path(@post) %>
```

Comment:
Can have a conditional here that only shows the edit link if the current user is the author of the post